include/orb/Orbit.h

```diff
@@ -325,8 +330,8 @@ class Orbit {
         r_ecef0= rel_r+ orb_r;
         v_ecef0= rel_v+ lin::cross(omega,orb_r);
         // step 5b rotate back to ecef
-        _recef= dcm_ecefhalf_ecef0*dcm_ecefhalf_ecef0*r_ecef0;
-        _vecef= dcm_ecefhalf_ecef0*dcm_ecefhalf_ecef0*v_ecef0;
+        _recef= dcm_ecefhalf_ecef0*(dcm_ecefhalf_ecef0*r_ecef0);
+        _vecef= dcm_ecefhalf_ecef0*(dcm_ecefhalf_ecef0*v_ecef0);
```

Comment:
Good efficient catch! Although, it'd be better if you forced a temporary to be created after the first matrix multiplication. Like:

    _recef= dcm_ecefhalf_ecef0*(dcm_ecefhalf_ecef0*r_ecef0).eval();

and similarly for `_vecef`. Without the eval call, each component of `dcm_ecefhalf_ecef0*r_ecef0` is calculated again every time it's required for the second matrix-vector multiplication.